redash/handlers/embed.py

```diff
@@ -89,7 +92,7 @@ def embed(query_id, visualization_id, org_slug=None):
             if results is None:
                 abort(400, message="Unable to get results for this query")
             else:
-                qr = {"data": json.loads(results)}
+                qr = results.to_dict()
```

Comment:
I was about to merge this, but then I realized -- we intentionally return only the data field to avoid leaking information via the query.

Can you change this into `project(results.to_dict(), ('data', 'retrieved_at'))` (you will need to import `project` from `funcy`)?